runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -166,6 +167,17 @@ public void testBossTask()
 		assertEquals(914, slayerPlugin.getPoints());
 	}
 
+	public void testNewPartnerTask()
+	{
+		Widget npcDialog = mock(Widget.class);
```

Comment:
This doesn't look right... The new message would be a chat message, not an NPC dialog.